protos/sources/vega/events/v1/events.proto

```diff
@@ -950,6 +950,19 @@ message VolumeDiscountProgramEnded {
   uint64 at_epoch = 4;
 }
 
+message PaidLiquidityFeesStats {
+  // Market the fees were paid in
```

Comment:
All of these descriptions need to end in a full-stop so that the docs-tool doesn't get confused and think they are titles.